cloud_platform_exoscale/models/cloud_platform.py

```diff
@@ -27,15 +29,81 @@ def _platform_kinds(self):
 
     @api.model
     def _config_by_server_env_for_exoscale(self):
+        fs_kinds = self._filestore_kinds()
         configs = {
-            'prod': PlatformConfig(filestore=FilestoreKind.s3),
-            'integration': PlatformConfig(filestore=FilestoreKind.s3),
-            'labs': PlatformConfig(filestore=FilestoreKind.s3),
-            'test': PlatformConfig(filestore=FilestoreKind.db),
-            'dev': PlatformConfig(filestore=FilestoreKind.db),
+            'prod': PlatformConfig(filestore=fs_kinds['s3']),
+            'integration': PlatformConfig(filestore=fs_kinds['s3']),
+            'labs': PlatformConfig(filestore=fs_kinds['s3']),
+            'test': PlatformConfig(filestore=fs_kinds['db']),
+            'dev': PlatformConfig(filestore=fs_kinds['db']),
         }
         return configs
 
     @api.model
-    def install_exoscale(self):
-        self.install('exoscale')
+    def _check_filestore(self, environment_name):
+        params = self.env['ir.config_parameter'].sudo()
+        use_s3 = (params.get_param('ir_attachment.location') ==
+                  S3_STORE_KIND.name)
+        if environment_name in ('prod', 'integration'):
+            # Labs instances use swift or s3 by default, but we don't want
```

Comment:
```suggestion
            # Labs instances use s3 by default, but we don't want
```
As we are now in exoscale, by default it will use only s3 correct?